lib/galaxy/tools/parameters/basic.py

```diff
@@ -163,9 +163,6 @@ def value_to_basic( self, value, app ):
         return self.to_string( value, app )
 
     def value_from_basic( self, value, app, ignore_errors=False ):
-        # HACK: Some things don't deal with unicode well, psycopg problem?
```

Comment:
This was introduced in commit 7bfc238304ac5e46abac83ff8eb222011ffe9dbe, but that hardly helps.